docs/setup_instructions/setup_overview.md

```diff
@@ -3,87 +3,35 @@ title: Pre-workshop setup
-2. [Install GitKraken](https://openscpca.readthedocs.io/en/latest/technical-setup/install-a-git-client/#install-gitkraken-on-windows-with-wsl-2) into the WSL 2 side of your computer
-    - Follow the instructions in the link above to also sign into GitKraken with your GitHub account
-3. [Install Miniconda](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/setup-conda/#install-conda), a lightweight distribution of the package management software conda, into the WSL 2 side of your computer
-    - You do _not_ need to also set up conda; we will do this together during the workshop
-4. [Install R and RStudio Server](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/install-r-rstudio/) into the WSL 2 side of your computer
-5. Install some R packages we will use in the workshop by running the following command in R Console:
-    ```r
-    install.packages(c("renv", "tidyverse", "rmarkdown", "here", "optparse"))
-    ```
-6. Install [Visual Studio Code (VS Code)](https://code.visualstudio.com/), a popular text editor, into the Windows side of your computer
-    - [Following these instructions](https://code.visualstudio.com/docs/remote/wsl#_getting-started), you should also install and turn on the [WSL 2 extension](https://code.visualstudio.com/docs/remote/wsl#_getting-started) to be able to use VS Code from WSL 2
+<!--
+To update the setup instructions, please modify one of the files rrp-setup.md or openrrp-setup.md.
+Do not modify this file!
+-->
 
+This will be a hands-on workshop, so to maximize our time, we would like you to arrive having completed a number of setup tasks, namely setting up accounts and installing software that we will be using.
 
+We will _not_ have time to do all of the setup on the day of the workshop and accomplish all of our goals for the workshop!
+Please be aware this setup process may take 1-2 hours to complete.
 
 
+**Table of contents**
 
-<!--
+* TOC goes here
+{:toc}
 
-Regular RRP installation instructions are commented out below.
+{% if site.workshop == "openrrp" %}
+### Only staying for "Reproducible Research Practices"?
 
-The main tasks are installation of R, RStudio, GitKraken, and a suitable text editor.
-For Windows users, we will also require the installation of Windows Subsystem for Linux (WSL).
-You may already have some of this software installed, but we encourage you to read through all of the instructions at the links below that pertain to your system.
+If you are only staying for the first half of the workshop and _not_ for the portion about OpenScPCA, you can follow these setup instructions.
+_Otherwise, please proceed to the next section and follow all other instructions._
+{% endif %}
 
-- [macOS installation instructions](mac_installation_instructions.md)
-- [Windows installation instructions](windows_installation_instructions.md)
+{% include_relative rrp-setup.md %}
```

Comment:
For simplicity, I would just put this content inline and not in a separate document